aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -196,8 +196,18 @@ impl AptosVMImpl {
         // The submitted transactions max gas units needs to be at least enough to cover the
         // intrinsic cost of the transaction as calculated against the size of the
         // underlying `RawTransaction`
-        let min_txn_fee =
-            gas_constants.to_external_units(calculate_intrinsic_gas(raw_bytes_len, gas_constants));
+        let min_txn_fee = {
+            let min_transaction_fee = gas_constants.min_transaction_gas_units;
```

Comment:
probably still want to have a function and explains why not use move one?

Reply:
Better than magic numbers